pkg/network/ebpf/c/protocols/http2/decoding-tls.h

```diff
@@ -812,7 +809,8 @@ int uprobe__http2_tls_headers_parser(struct pt_regs *ctx) {
         }
 
         http2_ctx->http2_stream_key.stream_id = current_frame.frame.stream_id;
-        current_stream = http2_fetch_stream(&http2_ctx->http2_stream_key);
+        // A new stream must start with a request, so if it does not exist, we should not process it.
+        current_stream = bpf_map_lookup_elem(&http2_in_flight, &http2_ctx->http2_stream_key);
```

Comment:
the change here is incorrect, we're in `uprobe__http2_tls_headers_parser` so here we can see "a new request" and we'll need to create a new stream for it